src/Core/Grid/Definition/Factory/FilterableEmployeeGridDefinitionFactory.php

```diff
@@ -0,0 +1,300 @@
+                                    [],
+                                    'Admin.Notifications.Warning'
+                                ),
+                                'route' => 'admin_employees_delete',
+                                'route_param_name' => 'employeeId',
+                                'route_param_field' => 'id_employee',
+                            ])
+                        ),
+                ])
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFilters()
+    {
+        return (new FilterCollection())
+            ->add(
+                (new Filter('id_employee', NumberType::class))
+                ->setTypeOptions([
+                    'required' => false,
+                    'attr' => [
+                        'placeholder' => $this->trans('Search ID', [], 'Admin.Actions'),
+                    ],
+                ])
+                ->setAssociatedColumn('id_employee')
+            )
+            ->add(
+                (new Filter('firstname', TextType::class))
+                ->setTypeOptions([
+                    'required' => false,
+                    'attr' => [
+                        'placeholder' => $this->trans('Search first name', [], 'Admin.Actions'),
+                    ],
+                ])
+                ->setAssociatedColumn('firstname')
+            )
+            ->add(
+                (new Filter('lastname', TextType::class))
+                ->setTypeOptions([
+                    'required' => false,
+                    'attr' => [
+                        'placeholder' => $this->trans('Search last name', [], 'Admin.Actions'),
+                    ],
+                ])
+                ->setAssociatedColumn('lastname')
+            )
+            ->add(
+                (new Filter('email', TextType::class))
+                ->setTypeOptions([
+                    'required' => false,
+                    'attr' => [
+                        'placeholder' => $this->trans('Search email', [], 'Admin.Actions'),
+                    ],
+                ])
+                ->setAssociatedColumn('email')
+            )
+            ->add(
+                (new Filter('profile', ProfileChoiceType::class))
+                ->setTypeOptions([
+                    'required' => false,
+                ])
+                ->setAssociatedColumn('profile')
+            )
+            ->add(
+                (new Filter('active', ChoiceType::class))
+                ->setTypeOptions([
+                    'choices' => [
+                        $this->trans('Yes', [], 'Admin.Global') => 1,
+                        $this->trans('No', [], 'Admin.Global') => 0,
+                    ],
+                    'required' => false,
+                    'choice_translation_domain' => false,
+                ])
+            )
+            ->add((new Filter('active', YesAndNoChoiceType::class))
+                ->setAssociatedColumn('active')
+            )
```

Comment:
Double active ? 🤔

Reply:
Nice catch @PululuK, this file is mainly just a copy/paste from [`EmployeeGridDefinitionFactory`](https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Grid/Definition/Factory/EmployeeGridDefinitionFactory.php) but indeed, I'll have to change it ;)

Reply:
@atomiix  see https://github.com/PrestaShop/PrestaShop/pull/19873

Reply:
@PululuK I removed this duplicated filter on the new `FilterableEmployeeGridDefinitionFactory` class and cherry-picked your commit https://github.com/PululuK/PrestaShop/commit/234d87e83633bc158ae1dffb69e45463dfbe2e8e We should be all good!

Reply:
Thank you @atomiix 

@matks  I'm very curious to see what will happen on my [PR](https://github.com/PrestaShop/PrestaShop/pull/19873) if you merge this one before 🤔

IMHO  :  

- We shouldn't fix `EmployeeGridDefinitionFactory.php` here. It's not in the scope of the issue. In this PR just `FilterableEmployeeGridDefinitionFactory.php` should be fixed
-  Or, you can close this [PR](https://github.com/PrestaShop/PrestaShop/pull/19873), but in this case QA team need to check if  `BO > Advanced params > teams` filter by active/disabled work too.